cmd/status/cmdstatus.go

```diff
@@ -59,7 +59,7 @@ type Runner struct {
 	Command    *cobra.Command
 	factory    cmdutil.Factory
 	invFactory inventory.ClientFactory
-	loader     manifestreader.ManifestLoader
+	loader     *InventoryLoader
```

Comment:
Shouldn't the type here be the Loader interface?

Reply:
@mortent Oh yes, it should be the Loader interface. I will change this and push a new one.